src/style.css

```diff
@@ -1,3 +1,51 @@
+html {
+  font-family:
+    system-ui,
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    Roboto,
+    Oxygen,
+    Ubuntu,
+    Cantarell,
+    'Open Sans',
+    'Helvetica Neue',
+    sans-serif;
+}
+
 body {
   margin: 0;
+  display: flex;
+  width: 40px;
+  height: 40px;
+  top: 10px;
+  left: 50px;
+}
+
+.nav {
+  line-height: 15px;
+}
+
+.header {
+  display: flex;
+  padding: 10px 20px;
+}
+
+.logo_img {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav_link {
+  width: 37px;
+  height: 4px;
+  top: 56px;
+  left: 509px;
+  border-radius: 8px;
+  margin-left: 40px;
+}
+
+:hover {
+  background-color: #00acdc;
```

Comment:
The `:hover` selector is not scoped to any specific element, which means it will apply to any element that is hovered over. Consider scoping it to a specific class or element, such as `.nav_link:hover`, to avoid unintended styling.